zksync/sdk/musig/schnorr_musig_signer.py

```diff
@@ -0,0 +1,89 @@
+from typing import List
+
+from zksync.sdk.musig.schnorr_musig_error import SchnorrMusigError
+from zksync.sdk.musig.schnorr_musig_native import *
+
+
+class SchnorrMusigSigner:
```

Comment:
Could be handy to add a `verify()` method here, too

Reply:
Does not make any sense, cause singer stores a single public key.
See comments: https://github.com/zksync-sdk/schnorr-musig-sdk-python/pull/1#discussion_r592140704